service/src/test/java/bio/terra/cbas/controllers/VerifyPactsRunSetsApiController.java

```diff
@@ -65,20 +72,80 @@ void before(PactVerificationContext context) {
     context.setTarget(new MockMvcTestTarget(mockMvc));
   }
 
+  @State({"initialize dependencies"})
```

Comment:
These state names could probably be more descriptive... like "Exactly one run set with ID ...." (since we're pinning it, we might as well make it easy to find?)

Reply:
good point, I will change that!